eventstore/whatsapp_actions.py

```diff
@@ -70,8 +74,7 @@ def handle_event(event):
 
 
 def handle_whatsapp_message_expired_error(event):
-    # TODO: handle whatsapp timeout system event
-    pass
+    handle_whatsapp_delivery_error(f"whatsapp:{event.recipient_id}")
```

Comment:
This should probably be called with `.delay`, so that it happens in the background, not in the request.